projects/ad469x_iio/app/app_config_stm32.h

```diff
@@ -0,0 +1,161 @@
+#define UART_MODULE                 5    // UART5
+#define UART_IRQ                    UART5_IRQn
+#define SPI_DEVICE_ID               1    // SPI1
+#define SPI_CS_PIN_NUM              15   // PA_15
+#define SPI_CS_PORT_BASE            GPIOA
+#define SPI_CS_PORT_NUM             0    // PORTA = 0
+#define CNV_PIN_NUM                 15   // PB_15
+#define CNV_PORT_NUM                1    // PORTB = 1
+#define CNV_PORT_BASE               GPIOB
+#define GP0_PIN_NUM                 7    // PG_7
+#define GP0_PORT_NUM                6    // PORTG = 6
+#define GP1_PIN_NUM                 10   // PG_10
+#define GP1_PORT_NUM                6    // PORTG = 6
+#define BSY_PIN_NUM                 GP0_PIN_NUM
+#define BSY_PORT_NUM                GP0_PORT_NUM
+#define RESET_PIN_NUM               9    // PG_9
+#define RESET_PORT_NUM              6    // PORTG = 6
+#define TRIGGER_INT_ID	            GP0_PIN_NUM
+#define TRIGGER_GPIO_PORT           0    // Unused macro
+#define TRIGGER_GPIO_PIN            GP0_PIN_NUM
+#define MAX_SPI_SCLK                22500000
+
+#define gpio_ops                    stm32_gpio_ops
+#define spi_ops		                stm32_spi_ops
+#define i2c_ops                     stm32_i2c_ops
+#define uart_ops                    stm32_uart_ops
+#define pwm_ops                     stm32_pwm_ops
+#define trigger_gpio_irq_ops        stm32_gpio_irq_ops
+#define dma_ops                     stm32_dma_ops
+#define trigger_gpio_handle         0    // Unused macro
+
+/* Timer specific macros used for calculating pwm
+ * period and duty cycle */
+#define TIMER_1_PRESCALER                  1
+#define TIMER_2_PRESCALER                  0
+#define TIMER_1_CLK_DIVIDER                2
+#define TIMER_2_CLK_DIVIDER                2
+#define TIMER_8_PRESCALER                  0
+#define TIMER_8_CLK_DIVIDER                2
+
+/* Timer Channels */
+#define TIMER_CHANNEL_1                    1
+#define TIMER_CHANNEL_2                    2
+#define TIMER_CHANNEL_3                    3
+
+#define TIMER1_ID                          1
+#define TIMER2_ID                          2
+#define TIMER8_ID                          8
+
+#define Rx_DMA_IRQ_ID               DMA2_Stream0_IRQn
+#define AD469x_TxDMA_CHANNEL_NUM    DMA_CHANNEL_7
+#define AD469x_RxDMA_CHANNEL_NUM    DMA_CHANNEL_3
+
+/* Define the max possible sampling (or output data) rate for a given platform.
+ * This is also used to find the time period to trigger a periodic conversion event.
+ * Note: Max possible ODR is 500KSPS per channel for burst/continuous data capture on
+ * IIO client. This is derived by testing the firmware on STM32F469NI MCU @22Mhz SPI clock.
+ * The max possible ODR can vary from board to board and data continuity is not guaranteed
+ * above this ODR on IIO oscilloscope
+ */
+#if (INTERFACE_MODE == SPI_INTERRUPT)
+#define SAMPLING_RATE					(62500)
+#define CONV_TRIGGER_DUTY_CYCLE_NSEC(x)	(x / 10)
+#else
+#define SAMPLING_RATE					   (500000)
```

Comment:
was this tested with 1msps? or only up to 500ksps?

Reply:
This has been tested only upto 500ksps